### PR TITLE
[FLINK-13679][chinese-translation]Translate "Code Style - Pull Requests & Changes" page into Chinese

### PR DESCRIPTION
**Pull request by LakeShen**
Translate "Code Style - Pull Requests & Changes" page into Chinese

### CODE REVIEW

**Comment by LakeShen** [comment]
Is there anyone to review it , thanks

File: contributing/code-style-and-quality-pull-requests.zh.md [hunk 1]
```diff
@@ -81,10 +81,10 @@ Examples for commit messages:
 * `[FLINK-11702][table-planner-blink] Introduce a new table type system`
 
 
-## 5. Changes to the observable behavior of the system
+## 5. 改变系统的可观察行为
+贡献者应该注意他们的 PRs 中的更改，这些改动会以任何方式破坏 Flink 的可观察行为，因为在许多情况下，这样的改动会破坏现有的配置。
+在编写代码或者对这个问题进行评论时，应该使用红色标记提出的问题，例如：
```
Comment (shuai-xu) [inline contributing/code-style-and-quality-pull-requests.zh.md]: 在编写代码或者审查这个问题时，应该使用红色标记来提出问题，例如

**Review by shuai-xu — COMMENTED** [review]
Hi @LakeShen , thanks for the contribution. I left some comments which should be addressed before merged.

**Comment by LakeShen** [comment]
Thank you to reivew it , shuai-xu, I will receive your comments.

**Comment by LakeShen** [comment]
Hi @shuai-xu , I have changed the translation according to your commits , if you have free time ,please review it , thank you .

**Comment by shuai-xu** [comment]
Hi @LakeShen , it seems that some comments are not addressed, it you have different ideas, you can comment to my suggestions.

**Comment by LakeShen** [comment]
Sure, I will check the translation once again. Thanks

**Comment by klion26** [comment]
@LakeShen What's the status of the PR?

**Comment by LakeShen** [comment]
@klion26 This pr is also open.

**Pull request closed without merging**